src/oas/computeOperations.ts

```diff
@@ -0,0 +1,29 @@
+import { IHttpOperation } from '@stoplight/types';
+import { flatten, get, keys, map } from 'lodash';
+import { OpenAPIObject } from 'openapi3-ts';
+import { Spec } from 'swagger-schema-official';
+
+import { HttpOperationTransformer } from '../oas/types';
+
+const methods = ['get', 'post', 'put', 'delete', 'options', 'head', 'patch', 'trace'];
+
+export function computeOasOperations(
```

Comment:
Nice, I like that it doesn't care that it's a oas2/3 document. Can we do something similar for the `transformOas2Operation` and `transformOas3Operation` and make a `trasnformOasOperation`?

Reply:
Is it required to pass in the transformers?

Can we just check the openapi or swagger property?